src/main/java/frc/robot/utils/HPStationTransfromUtil.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.utils;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+
+public class HPStationTransfromUtil {
+
+    private final double BASE_X_TRANSFORM_METERS = Units.inchesToMeters(10.6);
+    private final double BASE_Y_TRANSFORM_METERS = Units.inchesToMeters(14.6);
+
+    // The left or right HP station
+    public enum Station {
+        LEFT,
+        RIGHT
+    }
+
+    // Position on the HP station that the robot is going to
+    public enum Position {
```

Comment:
would describe where ONE/TWO is o the HP station

Reply:
assuming that they are each offset from the tag. would add an option that is exactly aligned to the tag